Sources/SwiftProtobuf/MessageExtension.swift

```diff
@@ -19,22 +19,22 @@
 /// pieces.
 public protocol MessageExtensionBase {
     var fieldNumber: Int { get }
-    var fieldNames: _NameMap.Names { get }
+    var _protobuf_fieldNames: _NameMap.Names { get }
     var messageType: Message.Type { get }
-    func newField() -> AnyExtensionField
+    func _protobuf_newField() -> AnyExtensionField
 }
 
 /// A "Message Extension" relates a particular extension field to
 /// a particular message.  The generic constraints allow
 /// compile-time compatibility checks.
 public class MessageExtension<FieldType: ExtensionField, MessageType: Message>: MessageExtensionBase {
     public let fieldNumber: Int
-    public var fieldNames: _NameMap.Names
+    public var _protobuf_fieldNames: _NameMap.Names
     public let messageType: Message.Type
     public let defaultValue: FieldType.ValueType
     public init(fieldNumber: Int, fieldNames: _NameMap.Names, defaultValue: FieldType.ValueType) {
```

Comment:
I think this is one of the few initializers we have that really is just for the generated code, do we want to do anything with the naming of this?